src/main/kotlin/com/quipalup/katydid/logentry/logentry/primaryadapter/httprest/CreateLogEntryEndpoint.kt

```diff
@@ -0,0 +1,36 @@
+package com.quipalup.katydid.logentry.logentry.primaryadapter.httprest
+
+import arrow.core.Either
+import arrow.core.flatMap
+import arrow.core.right
+import com.quipalup.katydid.logentry.application.CreateLogEntryCommand
+import com.quipalup.katydid.logentry.application.CreateLogEntryCommandHandler
+import com.quipalup.katydid.logentry.domain.LogEntryError
+import org.springframework.http.HttpStatus
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+internal class CreateLogEntryEndpoint(private val createLogEntryCommandHandler: CreateLogEntryCommandHandler) {
+    @PostMapping("/log-entries")
+    @ResponseStatus(HttpStatus.CREATED)
+    fun execute(): LogEntryResponseDocument {
+        return buildCreateRequest()
+            .flatMap { createLogEntryCommandHandler.execute(it) }
+            .fold(errorHandler()) { it }
+    }
+
+    private fun errorHandler(): (LogEntryError) -> LogEntryResponseDocument = { throw RuntimeException() }
```

Comment:
This is ok for now, in the near future we will refactor this in order to send a response with a formatted JSON body that is descriptive of the error and we'll also learn how to set a specific status code.